.gitignore

```diff
@@ -13,7 +13,6 @@ xcuserdata
 *.xccheckout
 *.moved-aside
 DerivedData
-*.hmap
```

Comment:
Xcode generates this `.hmap` and mostly might keeps it in the DerivedData folder I think. It may not keep it in the project sources directory.

Let me buid the example app in this repo and check if it overwrites this file

Reply:
Confirmed, Xcode keeps them in DerivedData and the .hmaps are not overwritten
<img width="1065" alt="Screenshot 2023-09-04 at 12 57 40 PM" src="https://github.com/gojekfarm/FLEX/assets/142002713/e89f6194-3fc0-44fe-8cb3-fa296aacacf2">
<img width="1001" alt="Screenshot 2023-09-04 at 12 58 19 PM" src="https://github.com/gojekfarm/FLEX/assets/142002713/c546ab55-9b97-4d53-9799-2cc906b1e550">